index.html

```diff
@@ -1,182 +1,185 @@
-            <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
-        </ul>
-        <ul class="skills">
-            <li id="title">Skills</li>
-            <li>Database Management</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Version Control</li>
-            <li>Web Development</li>
-            <img src="images/light.png" alt="light-bulb" class="bulb">
-        </ul>
+            <ul class="lang">
+                <li id="title">Languages</li>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <img src="images/ic_languages_mob.svg" alt="testimage" class="hashtag">
+            </ul>
+            <ul class="fram">
+                <li id="title">Frameworks</li>
+                <li>React.js</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <img src="images/Rectangle 18.png" alt="rectangle18" class="rectangle18">
+                <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
+            </ul>
+            <ul class="skills">
+                <li id="title">Skills</li>
+                <li>Database Management</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Version Control</li>
+                <li>Web Development</li>
+                <img src="images/light.png" alt="light-bulb" class="bulb">
+            </ul>
 
-          <div class="get-contact">
-            <img src="images/Shape (1).png" alt="image4" class="image4">
-            <img src="images/Shape (2).png" alt="image5" class="image5">
-            <h1>Get Started</h1>
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
-            <form class="contact-form">
-            <div>
-              <label id="name-label">Your name</label>
-              <br>
-              <input id="name"  type="text" placeholder="Ricardo"/>
-            </div>
-            <div>
-              <label id="email-label">Your email</label>
-              <br>
-              <input id="email" type="email" placeholder="rj_egas@..."/>
-            </div>
-            <div>
-              <label>Your message</label>
-              <br>
-              <textarea name="comments" id="comment-box" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+            <div class="get-contact">
+                <img src="images/Shape (1).png" alt="image4" class="image4">
+                <img src="images/Shape (2).png" alt="image5" class="image5">
+                <h1>Get Started</h1>
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
+                <form class="contact-form">
```

Comment:
- [ ] Per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), you are to collect data by using the [Formspree service](https://formspree.io/html) but this requirement hasn't been implemented yet so kindly include this as well :100: So you can receive data from users when the form is submitted :wink: 
- [ ] This `Form` element currently doesn't have a closing tag so kindly add a closing tag to it since the closing tag tells the web browser where the form ends, so it knows how to properly display the content. :+1: 
- [ ] Also, please ensure that all `input` fields have the `required` attribute to activate the validation on submission. To meet this requirement :+1: 

![image](https://user-images.githubusercontent.com/102699958/229938271-e70e13e4-7af6-42c2-a785-44fc0b1a8c22.png)